fix sentry alternatives

yuvraajsj18 opened the pull request:
- **wip**
- **fixed the formatting**
- **add opentelemetry CTA**
- **added key takeaway**
- **signoz vs newrelic (#1089)**
- **chore(deps): bump next from 14.2.10 to 14.2.21 (#1075)**

ankit01-oss commented on docs/tutorial/traefik-observablity.md:
```diff
@@ -20,21 +22,100 @@ of services running behind Traefik and troubleshoot issues.
+<Tabs>
+<TabItem value="signoz-cloud" label="SigNoz Cloud" default>
+
 In this section, we will see how to export Traefik metrics and traces to SigNoz.
```
```suggestion
In this section, we will see how to export Traefik metrics and traces to SigNoz cloud.
```

prashant-shahi replied:
seems like I have it the other way around.

prashant-shahi replied:
Done

ankit01-oss commented on docs/tutorial/traefik-observablity.md:
```diff
@@ -20,21 +22,100 @@ of services running behind Traefik and troubleshoot issues.
+      - --tracing.openTelemetry.grpc=true
+      - --tracing.openTelemetry.address=ingest.{region}.signoz.cloud:443
+      - --tracing.openTelemetry.insecure=false
+      - --tracing.openTelemetry.headers.signoz-access-token=SIGNOZ_INGESTION_KEY
```
Same. `// highlight-next-line` 

You can check more details on how to do this here: https://docusaurus.io/docs/markdown-features/code-blocks

prashant-shahi replied:
same

ankit01-oss commented on docs/tutorial/traefik-observablity.md:
```diff
@@ -20,21 +22,100 @@ of services running behind Traefik and troubleshoot issues.
+</TabItem>
+<TabItem value="self-host" label="Self-Host">
+
+In this section, we will see how to export Traefik metrics and traces to SigNoz Cloud.
```
```suggestion
In this section, we will see how to export Traefik metrics and traces to SigNoz.
```

ankit01-oss reviewed (commented):
<img width="713" alt="Screenshot 2023-08-30 at 8 20 13 PM" src="https://github.com/SigNoz/signoz.io/assets/83692067/1a917fbf-5d2f-4f3b-ab86-a53cc58d31f5">

In prerequisites, can we add one line:

If you don’t already have a SigNoz cloud account, you can sign up [here](https://signoz.io/teams/).

Left some other comments.

prashant-shahi commented on docs/tutorial/traefik-observablity.md:
```diff
@@ -20,21 +22,100 @@ of services running behind Traefik and troubleshoot issues.
+
+We will take an example of a simple `hello-app` running behind Traefik.
+
+_docker-compose.yaml_
```
Adding the following:

```md
We will take an example `docker-compose.yaml` with a simple `hello-app`
running behind Traefik.
```

prashant-shahi replied:
updated

prashant-shahi commented on docs/tutorial/traefik-observablity.md:
```diff
@@ -20,21 +22,100 @@ of services running behind Traefik and troubleshoot issues.
+      - --providers.docker
+      - --metrics.openTelemetry=true
+      - --metrics.openTelemetry.grpc=true
+      - --metrics.openTelemetry.address=ingest.{region}.signoz.cloud:443
```
Using `yaml {line-numbers}` instead since multiple lines need to be highlighted.

yuvraajsj18 committed:
wip

yuvraajsj18 committed:
signoz vs newrelic (#1089)

* page done

* add floating card

* Enhance SigNoz vs New Relic comparison page with improved metadata and SEO optimizations. Updated page title and description, added Open Graph and Twitter card data, and included relevant keywords. Adjusted internal link paths for better navigation.

* change floating card to bottom of the div to avoid h1 seo heading issue

* build

* updated theme to be consistent in all three pages

dependabot[bot] committed:
chore(deps): bump next from 14.2.10 to 14.2.21 (#1075)

Bumps [next](https://github.com/vercel/next.js) from 14.2.10 to 14.2.21.
- [Release notes](https://github.com/vercel/next.js/releases)
- [Changelog](https://github.com/vercel/next.js/blob/canary/release.js)
- [Commits](https://github.com/vercel/next.js/compare/v14.2.10...v14.2.21)

---
updated-dependencies:
- dependency-name: next
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>
Co-authored-by: dependabot[bot] <49699333+dependabot[bot]@users.noreply.github.com>

yuvraajsj18 committed:
fixed the formatting

yuvraajsj18 committed:
add opentelemetry CTA

yuvraajsj18 committed:
added key takeaway
